helpers/githubbot.py

```diff
@@ -96,11 +96,12 @@ def comment(self, pull_request_id, message, url, args, storage, coverage_diffs,
     self.post_comment(body, pr)
 
   def post_comment(self, body, pr):
-    past_comment = self.past_comment(pr)
-    if past_comment:
-      past_comment.edit(body)
-    else:
-      pr.create_issue_comment(body)
+    if self.constants.get('GH_COMMENT').lower() == 'true':
```

Comment:
Instead of checking this each time, maybe do `self.post_comments = constants.get('GH_COMMENT').lower() == 'true'` in `__init__` and just do the following here.

```python
def post_comment(self, body, pr):
if not self.post_comments:
  return

past_comment = self.past_comment(pr)
if past_comment:
  past_comment.edit(body)
else:
  pr.create_issue_comment(body)
```